docs/documentation/install/requirements.md

```diff
@@ -66,14 +66,13 @@ If it says another number such as `0.12` or `5.x.x`, you need to download and in
 
 #### Mac / Windows users
 
-Download version 14 from [nodejs.org](https://nodejs.org/en/).
+[Download version 14 from the official Node.js website.](https://nodejs.org/en/)
```

Comment:
small thing but why 'official' here and not in the other link? Does it add much?

Reply:
Fair point! You're right, it doesn't add much. Will remove.

```suggestion
[Download version 14 from the Node.js website.](https://nodejs.org/en/)
```